src/test/java/com/google/sps/BigBusinessesTest.java

```diff
@@ -134,16 +136,19 @@ private void setUpSampleDatabase() {
     String rating = "Rating";
     String photos = "Photos";
     datastore = DatastoreServiceFactory.getDatastoreService();
-    Entity businessEntity = new Entity("BigBusinesses");
+    Entity businessEntity;
+    Key key;
     for(Listing business: sampleDatabaseOfBigBusinesses) {
-      businessEntity = new Entity("BigBusinesses");
+      // key = KeyFactory.createKey("key",business.getName());
+      businessEntity = new Entity("BigBusinesses",business.getName());
       businessEntity.setProperty(title, business.getName());
```

Comment:
I think I should delete this line as I am now creating each business with a key associated with it's name. I want to get your opinion on this.